### PR TITLE
Wiki en seameet counterparts v1

### PR DESCRIPTION
**Pull request by itIsJim**

### CODE REVIEW

File: content/en/seameet/seameet-manual/13-seameet-google-doc.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+     parent: "seameet-manual"
+aliases:
+   - /en/seameet/13-seameet-google-doc/
+weight: 113
+toc: true
+---
+
+After the meeting ends, the meeting record will be exported as a Google Docs document. Click the button in the top right corner of the page to open Google Docs, which contains all the information from the just-concluded meeting.
+
+1. Bind Google Drive
```
Comment (xuchen) [inline content/en/seameet/seameet-manual/13-seameet-google-doc.md:19]: "Bind" is not a common word

File: content/en/seameet/seameet-manual/13-seameet-google-doc.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+   If you want to export the meeting record as a Google document, please bind the workspace to a Google Drive. You can bind Google Drive in **Workspace** - **Integration**.
+
+    <center>
+
+    <img src="/images/seameet-en/13-seameet-google-doc/seameet-bind-google-drive.png" alt="SeaMeet Bind Google Drive"/>
+
+    </center>
+
+2. Automatically Export Google Document After Meeting
```
Comment (xuchen) [inline content/en/seameet/seameet-manual/13-seameet-google-doc.md:29]: there should be a new line after the picture

File: content/en/seameet/seameet-manual/13-seameet-google-doc.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+   Click the Google Docs button in the top right corner of the meeting page to access the corresponding meeting record
+
+    <center>
+
+    <img src="/images/seameet-en/13-seameet-google-doc/seameet-auto-export-google-doc.png" alt="SeaMeet Auto Export Google Document"/>
+
+    </center>
```
Comment (xuchen) [inline content/en/seameet/seameet-manual/13-seameet-google-doc.md:37]: 1. do you have a screenshot of the google doc?
2. explain that the exported docs live in user's google drive and doesn't belong to seasalt.ai

**Review by xuchen — COMMENTED** [review]
quick comment: we changed "bot" to "SeaMeet Copilot" please replace all mentions of "bot"

File: content/en/seameet/seameet-manual/13-seameet-google-doc.md (hunk 3 above)
Comment (itIsJim) [reply content/en/seameet/seameet-manual/13-seameet-google-doc.md:39]: I put a screenshot of the downloaded screenshots. Please have a look and see if it is what we want.

**Comment by xuchen** [comment]
could you please remove: `14-seameet-advanced-dashboard` entirely. it's not relevant.

**Comment by xuchen** [comment]
Could you please re-organize the pages as follows (also do it for Chinese):

* SeaMeet User Guide
    - Quick Tutorial (merge "Getting Started" with "How to Use SeaMeet Copilot")
    - Workspace Management
        - Set Up SeaMeet Workspace
        - Team Workspace Management
    - General Setup
        - Switch Meeting Language
        - Auto Share Meeting Records
        - Auto Join Meetings
    - Meeting Copilot
        - View Meeting Records
        - Play Meeting Recordings
        - End Meeting Recording
        - Using Meeting Transcripts and AI Meeting Analytics
        - Meeting Labels
    - Integrations
        - Google Calendar
        - Google Drive (originally "Export Meeting to Google Doc")
        - Browser Extensions (talk about all browsers we support, and give the extension url)
    - How to Subscribe to a Plan
    - Provide Feedback
    - FAQ

**Review by xuchen — CHANGES_REQUESTED** [review]

File: content/en/seameet/seameet-manual/4-integration/0-browser-extension.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+        <p class="lead"></p>
+        <a class="btn btn-primary btn-lg px-4 mb-2" href="https://chrome.google.com/webstore/detail/seameet-ai-meeting-minute/gkkhkniggakfgioeeclbllpihmipkcmn" role="button">🚀 Install SeaMeet Extension</a>
+    </div>
+</div>
+
+<br/>
+
+SeaMeet is an easy-to-use meeting AI copilot that helps you transcribe, record, and analyze your meetings in REAL TIME.
+
+SeaMeet offers support to many different browsers, including Arc, Brave, and Edge. We recommend taking the first step by installing our browser extension, so you will always have SeaMeet on your side in meetings. Simply click the button above to install the SeaMeet extension.
```
Comment (xuchen) [inline content/en/seameet/seameet-manual/4-integration/0-browser-extension.md:27]: I got so frustrated with this...  Where's Chrome??? You shouldn't omit the most important browser we support please...

**Review by xuchen — APPROVED** [review]
this is in so much a better shape now, thanks!

**Pull request merged**